Makefile

```
@@ -41,6 +41,52 @@ check_tools:
+
+bench-docker: ## run cold-start harness inside python:3.13-slim (authoritative)
+	@echo "$(GREEN)bench-docker:$(NC) building measurement image and running harness"
+	@command -v $(DOCKER) >/dev/null 2>&1 || { echo >&2 "$(RED)$(DOCKER) is not installed. Aborting.$(NC)"; exit 1; }
+	@uv run python -m src.backend.tests.benchmarks.driver --mode docker
+
+bench-snapshot: ## one-shot: capture authoritative baseline and overwrite thresholds.json
+	@echo "$(GREEN)bench-snapshot:$(NC) capturing authoritative baseline + writing thresholds.json"
+	@echo "$(YELLOW)  NOTE:$(NC) authoritative numbers require a Linux GHA runner; local captures are for iteration only."
+	@uv run python -m src.backend.tests.benchmarks.snapshot
+
+bench-verify-synthetic: ## MEAS-08 prove the CI gate trips on a synthetic regression
```

checkmake (0.3.2)

[warning] Target body for "bench-verify-synthetic" exceeds allowed length of 5 lines (12).

(maxbodylength)